frontends/web/src/routes/account/info/buyReceiveCTA.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { route } from '../../../utils/route';
+import { Coin, IBalance } from '../../../api/account';
+import { Button } from '../../../components/forms';
+import { Balances } from '../summary/accountssummary';
+import styles from './buyReceiveCTA.module.css';
+
+type TBuyReceiveCTAProps = {
+  balanceList?: [string, IBalance][];
+  code?: string;
+  hideBuyButton?: boolean;
+  hideReceiveButton?: boolean;
+  unit?: string;
+};
+
+export const BuyReceiveCTA = ({ code, unit, hideBuyButton = false, hideReceiveButton = false, balanceList }: TBuyReceiveCTAProps) => {
+  const { t } = useTranslation();
+  const onBuyCTA = () => route(code ? `/buy/info/${code}` : '/buy/info');
+  const onReceiveCTA = () => {
+    if (balanceList) {
+      if (balanceList.length > 1) {
+        route('/receive');
```

Comment:
I find 2 components and 2 routes with the name receive a bit confusing.

how about `src/routes/account/receive/receive-select.tsx` and `account/receive-select` or similar?

![Screen Shot 2023-01-25 at 16 04 24](https://user-images.githubusercontent.com/546900/214598680-e4d9aa72-98b6-4d45-bb21-f3fe7855e491.png)

Reply:
Yeah I agree it's a bit confusing.

 What about `receive-select` for the new page w/ select and `account-receive` for the current receive page (the one for each account)?

Reply:
I think the file and the route for recieve.tsx  should stay as is, 

> What about receive-select for the new page w/ select and account-receive for the current receive page (the one for each account)?

for route or filename? I think both route and filename should use the same naming.

Reply:
Ok. Lets just use /receive-select for the new page orrr /accounts-receive. Then change the route accordingly.